command/volume_delete.go

```diff
@@ -78,8 +83,8 @@ func (c *VolumeDeleteCommand) Run(args []string) int {
 
 	// Check that we get exactly two arguments
 	args = flags.Args()
-	if l := len(args); l != 1 {
-		c.Ui.Error("This command takes one argument: <vol id>")
+	if l := len(args); l < 1 {
+		c.Ui.Error("This command takes at least one argument: <vol id>")
```

Comment:
Is this change necessary? I think `flags.Args()` returns only the arguments to the command, not flags. So the number of expected arguments is still 1 (`<vol id>`).